dcicutils/deployment_utils.py

```diff
@@ -802,6 +855,30 @@ def main(cls):
             sys.exit(1)
 
 
+class BasicCGAPIniFileManager(IniFileManager):
+    APP_KIND = 'cgap'
+
+
+class BasicLegacyCGAPIniFileManager(BasicCGAPIniFileManager):
+    APP_ORCHESTRATED = False
+
+
+class BasicOrchestratedCGAPIniFileManager(BasicCGAPIniFileManager):
+    APP_ORCHESTRATED = True
+
+
+class BasicFourfrontIniFileManager(IniFileManager):
+    APP_KIND = 'fourfront'
+
+
+class BasicLegacyFourfrontIniFileManager(BasicFourfrontIniFileManager):
+    APP_ORCHESTRATED = False
+
+
+class BasicOrchestratedFourfrontIniFileManager(BasicFourfrontIniFileManager):
+    APP_ORCHESTRATED = True
```

Comment:
Although it's clear reading the code, I would comment all of these describing what it is. For example
```
class BasicOrchestratedCGAPIniFileManager(BasicCGAPIniFileManager):
""" production.ini builder for CGAP environments in the Alpha infrastructure (4dn-cloud-infra) """
    APP_ORCHESTRATED = True
```

Reply:
I added doc strings.